notebooks/Media-Pretrained/01_Data_Layer.ipynb

```diff
@@ -94,25 +93,16 @@
     "\n",
     "To begin, we are going to use the latest MovieLens dataset, this dataset has over 25 million interactions and a rich collection of metadata for items. There is also a smaller version of this dataset, which can be used to shorten training times, while still incorporating the same capabilities as the full dataset.\n",
     "\n",
-    "Generally speaking your data will not arrive in a perfect form for Personalize, and will take some modification to be structured correctly. This notebook guides you through all of that. \n",
-    "\n",
-    "Set USE_FULL_MOVIELENS to True to use the full dataset."
-   ]
-  },
-  {
-   "cell_type": "code",
-   "execution_count": null,
-   "metadata": {},
-   "outputs": [],
-   "source": [
-    "USE_FULL_MOVIELENS = False"
+    "Generally speaking your data will not arrive in a perfect form for Personalize, and will take some modification to be structured correctly. This notebook guides you through all of that. "
```

Comment:
I suggest a change to the wording: This notebook guides you through that process.

Reply:
I have changed the text following your suggestion. Thank you.